model/ftn/wmesmfmd.ftn

```diff
@@ -789,6 +874,27 @@
       call ESMF_UtilIOUnitGet(idse); open(unit=idse, status='scratch');
       close(idsi); close(idso); close(idss); close(idst); close(idse);
 !
+! 1.g Get merging option for regional coupling that domians does not
+! overlap complately. This will blend the data coming from forcing with
+! the data coming from coupling.
+!
+      if (med_present) then
+        call NUOPC_CompAttributeGet(gcomp, name="merge_import", &
```

Comment:
This forces "merge_import" feature to only be available if med_present = .true.

Reply:
This is working without if statements. So, I removed it and can be used without activating mediator.